scvi/model/base/_utils.py

```diff
@@ -84,6 +86,20 @@ def _validate_var_names(adata, source_var_names):
         )
 
 
+def _attempt_backwards_compatible_load(module, model_state_dict):
+    # loading from 0.8 models
+    try:
+        new_model_state_dict = OrderedDict()
+        for key, value in model_state_dict.items():
+            new_key = key.replace(" ", "_")
+            new_model_state_dict.update({new_key: value})
+        module.load_state_dict(new_model_state_dict)
+    except RuntimeError:
+        return False
+
+    return True
+
+
```

Comment:
None of this will be necessary, there must've been a bad rebase somewhere and I just fixed FCLayers on master to be consistent with 0.8